.travis.yml

```diff
@@ -42,8 +42,6 @@ matrix:
     - os: linux
       env: PROJECT_DIR=examples/OpenCL TOOLCHAIN=gcc-4-8
     - os: linux
-      env: PROJECT_DIR=examples/OpenCL TOOLCHAIN=android-ndk-r10e-api-19-armeabi-v7a-neon
```

Comment:
Here is the link to the broken job:
* https://travis-ci.org/ingenue/hunter/jobs/180306953

Add it as a comment so if somebody want to take a look at the problem.